resource-managers/kubernetes/docker/src/main/dockerfiles/spark/entrypoint.sh

```diff
@@ -38,10 +38,10 @@ fi
 
 SPARK_K8S_CMD="$1"
```

Comment:
I see two possible pass-through conditions here: one is "empty SPARK_K8S_CMD" and the other is "SPARK_K8S_CMD is non empty but has non-spark command in it"  Is that the convention, or is the pass-through case always expected to be "empty SPARK_K8S_CMD" ?

Reply:
ack, I concur, this is the way the script works historically

Reply:
And it does the right thing w.r.t. `shift` in both cases?

Reply:
@erikerlandson yes, after a shift the leading arg is gone from "$@"